apps/web/composables/useAggregatedCountries/useAggregatedCountries.ts

```diff
@@ -0,0 +1,54 @@
+import { type AggregatedCountries } from '@plentymarkets/shop-api';
+import { type UseAggregatedCountriesReturn, UseAggregatedCountriesState, type GetAggregatedCountries } from './types';
+
+/**
+ * @description Composable for getting `AggregatedCountries`:
+ * - Active shipping countries
+ * - EU geo-regulated countries
+ *
+ * @example
+ * ``` ts
+ * const {
+ *  data,
+ *  loading,
+ *  useGeoRegulatedCountries,
+ *  getAggregatedCountries,
+ *  getDefaultCountries,
+ *  getGeoRegulatedCountries,
+ * } = useAggregatedCountries();
+ * ```
+ */
+export const useAggregatedCountries: UseAggregatedCountriesReturn = () => {
+  const state = useState<UseAggregatedCountriesState>('useAggregatedCountries', () => ({
+    data: {} as AggregatedCountries,
+    loading: false,
+  }));
+
+  const useGeoRegulatedCountries = state.value.data?.geoRegulated?.length > 0;
+
+  const getAggregatedCountries: GetAggregatedCountries = async () => {
```

Comment:
```suggestion
  const fetchAggregatedCountries: GetAggregatedCountries = async () => {
```

Reply:
I find it confusing with the getters for `getDefaultCountries` `getGeoRegulatedCountries` so calling the api getter fetch would make it simpler to understand imo